src/GovUk.Education.ExploreEducationStatistics.Content.Model/Release.cs

```diff
@@ -225,6 +225,10 @@ public PartialDate NextReleaseDate
                 {
                     _nextReleaseDate = value;
                 }
+                else if (value.IsEmpty())
+                {
+                    _nextReleaseDate = null;
+                }
```

Comment:
What's this change for?

Reply:
Oh yeah, forgot about this.  A bug that Nusrath and I spotted when writing the UI tests.  You couldn't unset a Next Release Date during a Release status update if it had already been set to something once.